### ISSUE TITLE
UI is blinking when trying to expand a step which has no configuration needed

### ISSUE THREAD

**Issue by apupier** [syndesisio/syndesis-ui#1167]
- create an integration using a Twitter mention connection
- click on the arrow to expand/configure the Twitter Mention step
--> UI is blinking to get back to previous state

see https://drive.google.com/open?id=0B1cUBNButvV9RnNLUjRueFRLTnc

Suggestions:
- remove the '>' when there is no configuration needed
- or provide a page telling user that there is no parameter to configure but that provides a description of the connection for instance

**Comment by gashcrumb** [comment]
Yeah, this is something we need to figure out for both the front and back end.  Currently we only know that there's no configuration for the step when that page loads up, not before, so what you see is the UI requesting the config form from the backend, the response comes back that there's no form and we currently punt you back to the save or add step page.

@zregvart wonder, could we have some kind of flag on the action if it has no configuration?  That'd help here...
